app/code/Magento/InventorySales/Model/OrderRepository.php

```diff
@@ -0,0 +1,227 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\InventorySales\Model;
+
+use Magento\Framework\Api\ExtensionAttribute\JoinProcessorInterface;
+use Magento\Framework\Api\SearchCriteria\CollectionProcessorInterface;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Serialize\Serializer\Json;
+use Magento\Framework\Serialize\Serializer\Json as JsonSerializer;
+use Magento\Payment\Api\Data\PaymentAdditionalInfoInterface;
```

Comment:
We don't want to depend on `Payment` here. Should look for ways to eliminate it.